parlai/agents/hugging_face/README.md

```diff
@@ -24,5 +24,26 @@ Enter Your Message: Parrots are
 parlai train_model -m hugging_face/gpt2 --add-special-tokens True --add-start-token True --gpt2-size medium -t convai2 -bs 2 -mf <modelfile>
 ```
 
-## Other models
-_Other models are coming soon -- stay tuned!_
\ No newline at end of file
+## DialoGPT
+
+To use GPT2, run your command with the flag: `-m hugging_face/dialogpt`.
+
+### Examples
+**Talk to DialoGPT large in interactive mode, with beam size 10, 3-gram beam blocking, and minimum beam length 25:**
+```bash
+parlai interactive -m hugging_face/dialogpt --add-special-tokens False --gpt2-size large --inference beam --beam-size 10 --beam-context-block-ngram 3 --beam-block-ngram 3 --beam-min-length 25
+```
+_Note:_ In the above command, we must have the flag `--add-special-tokens False` if we want to use the model _without_ finetuning it.
+
+Here is example output from the above command:
+```
+Enter Your Message: What do you think of parrots?
+[Gpt2]:  I love parrots. They are the best. I love them so much. I wish I had a pet parrot.
+```
+
+
+**Fine-tune DialoGPT medium on the ConvAI2 task:**
+```bash
+parlai train_model -m hugging_face/dialogpt --add-special-tokens True --delimiter '\n' --add-start-token True --gpt2-size medium -t convai2 -bs 2 -mf <modelfile>
```

Comment:
@EricMichaelSmith didn't you have issues specifying  `--delimiter '\n'` on the command line?

Reply:
> @EricMichaelSmith didn't you have issues specifying `--delimiter '\n'` on the command line?

Yeah @emilydinan I had to do `--delimiter $'\n'` to get the newline to be set correctly. See the first response to https://unix.stackexchange.com/questions/20035/how-to-add-newlines-into-variables-in-bash-script